tests/lib/outputs/csv/csv_test.py

```diff
@@ -0,0 +1,173 @@
+        assert output_data["STATUS_EXTENDED"] == "Extended status"
+        assert output_data["MUTED"] is False
+        assert output_data["SERVICE_NAME"] == "Example Service"
+        assert output_data["SUBSERVICE_NAME"] == "Example Subservice"
+        assert output_data["SEVERITY"] == "critical"
+        assert output_data["RESOURCE_TYPE"] == "Instance"
+        assert output_data["RESOURCE_UID"] == "resource-123"
+        assert output_data["RESOURCE_NAME"] == "Example Resource"
+        assert (
+            output_data["RESOURCE_DETAILS"] == "Detailed information about the resource"
+        )
+        assert output_data["RESOURCE_TAGS"] == "tag1,tag2"
+        assert output_data["PARTITION"] == "aws"
+        assert output_data["REGION"] == "us-west-1"
+        assert output_data["DESCRIPTION"] == "Description of the finding"
+        assert output_data["RISK"] == "High"
+        assert output_data["RELATED_URL"] == "http://example.com"
+        assert output_data["REMEDIATION_RECOMMENDATION_TEXT"] == "Recommendation text"
+        assert (
+            output_data["REMEDIATION_RECOMMENDATION_URL"]
+            == "http://example.com/remediation"
+        )
+        assert output_data["REMEDIATION_CODE_NATIVEIAC"] == "native-iac-code"
+        assert output_data["REMEDIATION_CODE_TERRAFORM"] == "terraform-code"
+        assert output_data["REMEDIATION_CODE_CLI"] == "cli-code"
+        assert output_data["REMEDIATION_CODE_OTHER"] == "other-code"
+        assert output_data["COMPLIANCE"] == "compliance_key: compliance_value"
+        assert output_data["CATEGORIES"] == "category1,category2"
+        assert output_data["DEPENDS_ON"] == "dependency"
+        assert output_data["RELATED_TO"] == "related finding"
+        assert output_data["NOTES"] == "Notes about the finding"
+        assert output_data["PROWLER_VERSION"] == "1.0"
+
+    def test_csv_write_to_file(self):
+        mock_file = StringIO()
+        findings = [self.generate_finding()]
+
+        output = CSV(findings)
+        output._file_descriptor = mock_file
+
+        output.batch_write_data_to_file()
+
+        mock_file.seek(0)
+        content = mock_file.read()
+        content = content.split("PROWLER_VERSION")[1]
+        content = content.split("1.0")[0]
+        content = content + "1.0;"
+        string = ""
+        for key, value in output.data[0].items():
+            if "Status." in str(value) or "Severity." in str(value):
+                value = str(value).split(".")[1]
+            string += f"{value};"
+        assert string in content
+
+    def test_abstract_methods(self):
+        class DummyOutput(Output):
+            def transform(self, finding: Finding):
+                pass
+
+            def batch_write_data_to_file(self):
+                pass
+
+        dummy_output = DummyOutput(self.generate_finding())
+        assert dummy_output.transform(self.generate_finding()) is None
+        assert dummy_output.batch_write_data_to_file() is None
```

Comment:
What's the value of this? I'm not getting the point of asserting that to `None`. I think you want to check that the abstract method creation is enforced right?